content/400-reference/200-api-reference/300-environment-variables-reference.mdx

```diff
@@ -94,29 +94,43 @@ HTTP_PROXY=http://proxy.example.com
 HTTPS_PROXY=https://proxy.example.com
 ```
 
-## Binary environment variables
+## Engine environment variables
+
+###
+
+#### PRISMA_CLI_QUERY_ENGINE_TYPE
+
+binary|library
+
+#### PRISMA_CLIENT_ENGINE_TYPE
+
+binary|library
```

Comment:
New switching logic, needs link to section that actually describes that. (query engine page maybe?)